### PR TITLE
Portfolio-Desktop-Version

### PR DESCRIPTION
**Pull request by karam084**
- Create index.html file
- Create styles.css file
- Add media icons folder
- Add shapes icons folder
- Add src-image folder
- Add linter file
- Create first work card
- Create a second work card
- Create third and fourth work cards
- Create a form card about me
- Create a contact form 
- Add media queries

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @karam084, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Great job with GitHub flow
:heavy_check_mark: Good job passing linter checks

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below under the review._

Please also consider:
- [x] Please make sure the author's(your) information is in one place in the readme file as this will make your readme file look cleaner and professional. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+            >My Logo</a
+          >
+          <ul class="nav-title">
+            <li class="title1">Portoflio</li>
+            <li class="title2">About</li>
+            <li class="title3">Contact</li>
+          </ul>
+        </nav>
+      </header>
+      <section class="content-name">
```
[the remark this answers is not in the document]
Comment (D4nielJ) [reply index.html:50]: - [ ] As requested by the last reviewer, your mobile version is missing from the project. Although your desktop design is almost finished and only some details need more work. It is indispensable that you're mobile version works. That's the main point of using media queries to transform a mobile design, onto a desktop design. Kindly please fix this issue.

If you need to, you might want to take a step back, save your desktop CSS in another file and use the mobile design that you had already implemented. Then start placing the desktop CSS in your media queries again making sure that your mobile design remains intact. [You can read more on the subject in the provided lessons from Microverse.](https://github.com/microverseinc/curriculum-html-css/blob/main/responsivness.md)

image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @karam084, 

Great work on making the changes requested by the previous reviewer(s) 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Please kindly fix those.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @karam084  Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:


brent-rambo-thumbs-up


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**